src/links/links.macros.html

```diff
@@ -8,18 +8,29 @@
 {% from '../icons/icons.macros.html' import icon as icon_macro %}
 
 {#
+  * Renders a link
+  * @param {string} content - Link text content
+  * @param {string} href - Path or full URL for link
+  * @param {string} [class] - Class name for style or JS targeting
+  * @param {string} [id] - ID for JS targeting
+#}
+{% macro link(content, href, class=null, id=null) %}
+<a class="link{% if class %} {{ class }}{% endif %}"{% if id %} id="{{ id }}"{% endif %} href="{{ href }}">
+  {{ content }}
+</a>
+{% endmacro %}
```

Comment:
I've found from past projects that having a granular macro such as link often created more headache than writing the code. We didn't have content in yaml though so that could be the problem. 

It is fine to keep this as sometimes using the macro make keep code more readable!